troposphere/static/js/models/Instance.js

```diff
@@ -6,23 +6,16 @@ import InstanceState from "./InstanceState";
 
 export default Backbone.Model.extend({
```

Comment:
If there is no `start_date` I want an error to be logged. It's not an error we can recover from. This code isn't doing validation, its just transforming properties. Validation occurs in another place.